c2cgeoform/views/abstract_views.py

```diff
@@ -463,11 +490,11 @@ def _item_actions(self, item: T, readonly: bool = False) -> list[ItemAction]:
                 )
             )
 
-        if inspect(item).persistent and not readonly:  # type: ignore[union-attr]
+        if inspected_item.persistent and not readonly:
             actions.append(
                 ItemAction(
                     name="delete",
-                    label=_("Delete"),
+                    label=_("DeleteResponse"),
```

Comment:
This is the label of the button, not a python type.

Reply:
Oups thanks :-)